hosted_control_planes/hcp-import.adoc

```diff
@@ -8,6 +8,15 @@ toc::[]
 
 Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available.
 
+include::modules/hcp-import-limitations.adoc[leveloffset=+1]
+
+[role="_additional-resources_{context}"]
+== Additional resources
+
+* xref:../hosted_control_planes/hcp-updating.adoc#hcp-update-node-pools_hcp-updating[Updating node pools in a hosted cluster]
+* xref:../hosted_control_planes/hcp-updating.adoc#hcp-update-ocp-hc_hcp-updating[Updating a control plane in a hosted cluster]
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#discover-hosted-acm[Discovering {mce} hosted clusters in {rh-rhacm-title}]
```

Comment:
should this point to 2.12 instead of 2.11?

maybe a different thread, but since this is related to hcp, does it actually belong in the hcp docs instead?

Reply:
2.12 docs aren't published yet so added 2.11 link. But I can update once 2.12 is GA. 

The section is inclined more towards ACM hence I believe it belongs in the ACM docs.